qa/integration-tests-engine/src/test/java/org/camunda/bpm/integrationtest/functional/spin/FeelEngineTest.java

```diff
@@ -17,6 +17,7 @@
 package org.camunda.bpm.integrationtest.functional.spin;
```

Comment:
Renamed test file as it was not picked up by the test runner. All other test file in package also has the `Test` postfix.

Reply:
I check that other tests are picked up so it was only this missing.